lib/smart_answer_flows/uk-benefits-abroad.rb

```diff
@@ -601,14 +592,151 @@ def define
         end
       end
 
+      # Going abroad
+      radio :worked_in_eea_or_switzerland? do
+        option :before_jan_2021
+        option :after_jan_2021
+        option :no
+
+        next_node do |response|
+          case response
+          when "before_jan_2021"
+            if calculator.benefit == "jsa"
+              outcome :jsa_eea_going_abroad_maybe_outcome
+            elsif calculator.benefit == "winter_fuel_payment"
+              outcome :wfp_going_abroad_eea_maybe_outcome
+            elsif calculator.benefit == "esa"
+              if calculator.going_abroad
+                outcome :esa_going_abroad_eea_outcome
+              else
+                outcome :esa_already_abroad_eea_outcome
+              end
+            elsif calculator.benefit == "disability_benefits"
+              if calculator.going_abroad
+                outcome :db_going_abroad_eea_outcome
+              else
+                outcome :db_already_abroad_eea_outcome
+              end
+            end
+          when "after_jan_2021", "no"
+            question :parents_lived_in_eea_or_switzerland?
+          end
+        end
+      end
+
+      radio :parents_lived_in_eea_or_switzerland? do
+        option :before_jan_2021
+        option :after_jan_2021
+        option :no
+
+        next_node do |response|
+          case response
+          when "before_jan_2021"
+            if calculator.benefit == "jsa"
+              outcome :jsa_eea_going_abroad_maybe_outcome
+            elsif calculator.benefit == "winter_fuel_payment"
+              outcome :wfp_going_abroad_eea_maybe_outcome
+            elsif calculator.benefit == "esa"
+              if calculator.going_abroad
+                outcome :esa_going_abroad_eea_outcome
+              else
+                outcome :esa_already_abroad_eea_outcome
+              end
+            elsif calculator.benefit == "disability_benefits"
+              if calculator.going_abroad
+                outcome :db_going_abroad_eea_outcome
+              else
+                outcome :db_already_abroad_eea_outcome
+              end
+            end
+          when "after_jan_2021"
```

Comment:
```suggestion
          when "after_jan_2021", "no"
```

Then remove lines 670-689 (unless I'm wrong about them being identical!).